typescript/introduction/tsc-using-types/tsc-types-in-functions.md

```diff
@@ -35,7 +35,7 @@ hello(123);
 // error: Argument of type 'number' is not assignable to parameter of type 'string'.
 ```
 
-TypeScript will also throw an error if the wrong number of arguments[1] is passed in:
+TypeScript will also throw an error if the wrong number of [arguments](https://enki.com/glossary/general/parameter-vs-argument.md) is passed in:
```

Comment:
should add the glossary file in this pr as well

Reply:
also, the `.md` extension should not be included when linking in the insights, i.e. `/parameter-vs-argument` instead of `/parameter-vs-argument.md`

Reply:
oh, the entry is there. the problem is most likely the `.md`